packages/shared/src/components/modals/UpvotedPopupModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactElement } from 'react';
+import { ResponsiveModal } from './ResponsiveModal';
+import { ModalProps } from './StyledModal';
+import { ModalCloseButton } from './ModalCloseButton';
+import { UpvoterList, UpvoterListProps } from '../profile/UpvoterList';
+import {
+  UpvoterListPlaceholder,
+  UpvoterListPlaceholderProps,
+} from '../profile/UpvoterListPlaceholder';
+
+export interface UpvotedPopupModalProps extends ModalProps {
+  listProps: UpvoterListProps;
+  listPlaceholderProps?: UpvoterListPlaceholderProps;
+}
+
+export function UpvotedPopupModal({
+  listProps,
+  listPlaceholderProps = {},
+  onRequestClose,
+  children,
+  ...modalProps
+}: UpvotedPopupModalProps): ReactElement {
+  return (
+    <ResponsiveModal
+      {...modalProps}
+      onRequestClose={onRequestClose}
+      style={{
+        content: {
+          position: 'relative',
+          padding: 0,
+          maxHeight: '40rem',
+          maxWidth: '30rem',
+          overflow: 'hidden',
```

Comment:
Why is that?

Reply:
This is so the header won't have the scrollbar, it will have to start on the actual content.

Reply:
We can probably make this better by adding a `title` property on the modal which will immediately have a `header`  section (including border and close button), then a `main` section which will be the content of the `children` props. What do you think?